components/image-dropzone/image-dropzone.js

```diff
@@ -0,0 +1,49 @@
+import { getLibs } from '../../scripts/utils.js';
+import { styles } from './image-dropzone.css.js';
+import { getIcon } from '../../utils/utils.js';
+const { createTag } = await import(`${getLibs()}/utils/utils.js`);
+
+
+class ImageDropzone extends HTMLElement {
```

Comment:
Can we chat again next week on this level of extractions? I would like to understand better how you are envisioning the complete integration of a dropzone with this and maybe we should try to make more headways on some other more concrete deliverables before we commit to this work.

I also want to be mindful that the more we extract, the more we start to take away from each component's own ability to handle things differently. The idea of these components is that we can do very specific things based on what form component it is.

For example, the img dropzone for event hero and thumbnail image are immediate uploads -- upload as the drag&drop happens, while the speaker / host dropzone are uploaded as separate submit for each speaker / host. 

With the repeater, It just occurred to me that we might also still have a few pieces missing:

- The repeater needs to cleverly re-index each of the cloned elements with an ID attribute.
- The repeater function needs to receive a callback to reattach any of the lost eventListeners from the process of cloning.